cosmos/client.go

```diff
@@ -47,8 +52,19 @@ func (c *Client) Query(path string, data cmn.HexBytes, ptr interface{}) error {
 }
 
 // BuildAndBroadcastMsg builds and signs message and broadcast it to node.
-func (c *Client) BuildAndBroadcastMsg(msg sdktypes.Msg, accName, accPassword string, accNumber, accSeq uint64) (*abci.ResponseDeliverTx, error) {
-	txBuilder := NewTxBuilder(c.cdc, accNumber, accSeq, c.kb, c.chainID)
+func (c *Client) BuildAndBroadcastMsg(msg sdktypes.Msg, accName, accPassword string, accNumber uint64) (*abci.ResponseDeliverTx, error) {
+	info, err := c.kb.Get(accName)
+	if err != nil {
+		return nil, err
+	}
+
+	acc, err := auth.NewAccountRetriever(c).GetAccount(info.GetAddress())
```

Comment:
could use `GetAccountNumberSequence` instead

Reply:
I didn't spot that method. Of course we can use.